### PR TITLE
Update sample 1-2  to follow basher and zero trust

### PR DESCRIPTION
**Pull request by salman90**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ x ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ x ] Feature
[ ] Code style update (formatting, local variables)
[ x ] Refactoring (no functional changes, no api changes)
[ x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 1-Authentication/2-sign-in-b2c/AppCreationScripts/sample.json [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
-      "Endpoint": "AAD v2.0",
-      "Provider": "B2C"
+        "Title": "React single-page application using MSAL React to sign-in users against Azure AD B2C",
+        "Level": 100,
+        "Client": "React SPA",
+        "RepositoryUrl": "ms-identity-javascript-react-tutorial",
+        "Endpoint": "AAD v2.0",
+        "Languages": ["javascript"],
+        "Description": "React single-page application using MSAL React to sign-in users against Azure AD B2C",
+        "Products": ["azure-active-directory-b2c", "msal-js"],
```
Comment (derisen) [inline 1-Authentication/2-sign-in-b2c/AppCreationScripts/sample.json:10]:
```suggestion
        "Products": ["azure-active-directory-b2c", "msal-js", "msal-react"],
```

File: 1-Authentication/2-sign-in-b2c/README.md [hunk 2]
```diff
@@ -1,17 +1,35 @@
+---
+page_type: sample
+name: React single-page application using MSAL React to sign-in users against Azure AD B2C
+description: React single-page application using MSAL React to sign-in users against Azure AD B2C
+languages:
+ - javascript
+products:
+ - azure-active-directory-b2c
+ - msal-js
```
Comment (derisen) [inline 1-Authentication/2-sign-in-b2c/README.md:9]:
```suggestion
 - msal-js
 - msal-react
```

File: 1-Authentication/2-sign-in-b2c/README.md [hunk 3]
```diff
@@ -1,17 +1,35 @@
...
+urlFragment: ms-identity-javascript-react-tutorial
+extensions:
+- services: ms-identity
+- platform: JavaScript
+- endpoint: AAD v2.0
+- level: 100
+- client: React SPA
+- service:
```
Comment (derisen) [inline 1-Authentication/2-sign-in-b2c/README.md:17]:
```suggestion
```

**Comment by derisen** [comment]
@salman90 looks like missing the account switch feature, let's also add it

File: 1-Authentication/2-sign-in-b2c/README.md [hunk 4]
```diff
@@ -24,97 +42,107 @@ Here you'll learn about authentication and **B2C** concepts, such as [ID tokens]
...
 
-### Create User Flows and Custom Policies
+#### Create User Flows and Custom Policies
+
+Please refer to: [Tutorial: Create userflows in Azure Active Directory B2C](https://docs.microsoft.com/azure/active-directory-b2c/tutorial-create-user-flows)
 
-Please refer to: [Tutorial: Create user flows in Azure Active Directory B2C](https://docs.microsoft.com/azure/active-directory-b2c/tutorial-create-user-flows)
+> :warning: You may want additional claims (such as **username**, **emails**, **object ID** (*oid*) and etc.) to appear in the ID tokens obtained from Azure AD B2C user-flows. In that case, please refer to [User profile attributes](https://learn.microsoft.com/azure/active-directory-b2c/user-profile-attributes) to learn about how to configure your user-flows to emit those claims. For instance, to emit the **username** claim, navigate to the [Azure portal](https://portal.azure.com) and locate the **Azure AD B2C** service. Then, navigate to the **User flows** blade. Select the **User Attributes** tab and make sure **Email Address** is checked. Then select the **Application Claims** tab and make sure **Email Addresses** is checked.
```
Comment (kalyankrishna1) [inline 1-Authentication/2-sign-in-b2c/README.md:105]: reword as discussed

**Review by kalyankrishna1 — APPROVED** [review]
:shipit:

File: 1-Authentication/2-sign-in-b2c/README.md (hunk 4 above)
Comment (derisen) [inline 1-Authentication/2-sign-in-b2c/README.md:105]:
```suggestion
> :warning: This sample requires Azure AD B2C to emit the **emails** claim in the ID token, which is used as **username** by MSAL. To do so, navigate to the [Azure portal](https://portal.azure.com) and locate the **Azure AD B2C** service. Then, navigate to the **User flows** blade. Select the **User Attributes** tab and make sure **Email Address** is checked. Then select the **Application Claims** tab and make sure **Email Addresses** is checked. 
>
> You may want additional claims (such as  **object ID** (*oid*) and etc.) to appear in the ID tokens obtained from Azure AD B2C user-flows. In that case, please refer to [User profile attributes](https://learn.microsoft.com/azure/active-directory-b2c/user-profile-attributes) to learn about how to configure your user-flows to emit those claims. 
```

**Comment by salman90** [comment]
> @salman90 looks like missing the account switch feature, let's also add it

Let's work on the account selection in a separate PR.

**Pull request merged**